cmd/keytransparency-server/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM golang
+
+ENV DB_USER=test \
+    DB_PASSWORD=zaphod \
+    DB_DATABASE=test \
+    DB_HOST=127.0.0.0:3306
```

Comment:
The mysql db image uses the following environment variables to setup and configure the database on initial startup. 

```
    environment:
      MYSQL_PASSWORD: zaphod
      MYSQL_USER: test
      MYSQL_DATABASE: test
      MYSQL_RANDOM_ROOT_PASSWORD: "yes"
```